CHANGELOG.md

```diff
@@ -2,6 +2,85 @@
 
 All notable changes to this project will be documented in this file.
 
+## [1.0.0] - 2023-09-18
+
+### Bug Fixes
+
+- Crate can't contain broken symlink
+- Remove executable flag from fish completion file
+- Use proc_mounts only on linux
+- Hotfix harmful documentation
+- Fix hyperlinks on Windows
+- Nix flake check also builds the package
+- [**breaking**] Change number_huge and unit_huge to match the man page short codes
+
+### Documentation
+
+- Added cafkafk suggestions
+- Fix codeblocks in zsh completions
+- Update README.md
+- Document filetypes theme and rename trait
+- Link directly to space
+- Add Mac support for the --mount option in the man page
+- Add SAFETY comments to unsafe code blocks
+- Update deb instructions to use keyring
+- Fix chmod in deb installation instructions
+- Add potential gpg install to deb installation instructions
+- Document character style pairs in the code and match with man page
+- Add install instructions for Void Linux
+- Documentation of 'sn' and 'sb' conflicted with later docs
+- Document dimmed and italic style codes
+
+### Features
+
+- Add completion files in deb packaging script
+- Adds filtering for Windows hidden files
+- Support --mount option on Mac
+- Support --mount option on Mac
```

Comment:
So good it's there twice? 😄

Reply:
Well it is good! fixed it thou, nice catch